src/downloader.rs

```diff
@@ -81,8 +84,7 @@ impl RoutingMessageHandler for GossipRouter {
 	fn handle_channel_update(&self, msg: &ChannelUpdate) -> Result<bool, LightningError> {
 		let output_value = self.native_router.handle_channel_update(msg)?;
 
-		let mut counter = self.counter.write().unwrap();
-		counter.channel_updates += 1;
+		self.counter.write().unwrap().channel_updates += 1;
```

Comment:
is this gonna result in an immediate drop following the addition, without requiring an explicit scope?

Reply:
sorry, just double-checking